.github/workflows/linux-tests.yml

```diff
@@ -22,21 +22,19 @@ jobs:
       matrix:
         PYTHON_VERSION: ['3.7', '3.8', '3.9']
         INSTALL_TYPE: ['conda', 'pip']
+        QT_LIB: ['pyqt5', 'pyqt6']
+        exclude:
+          - INSTALL_TYPE: conda
+            QT_LIB: pyqt6
     steps:
       - name: Checkout branch
-        uses: actions/checkout@v1
-      - name: Install System Packages (conda)
-        if: matrix.INSTALL_TYPE == 'conda'
-        run: |
-          sudo apt-get update
-          sudo apt-get install libegl1-mesa
-      - name: Install System Packages (pip)
-        if: matrix.INSTALL_TYPE == 'pip'
+        uses: actions/checkout@v3
+      - name: Install System Packages
         run: |
           sudo apt-get update
-          sudo apt-get install -qq pyqt5-dev-tools
+          sudo apt-get install -y --no-install-recommends '^libxcb.*-dev' libx11-xcb-dev libglu1-mesa-dev libxrender-dev libxi-dev libxkbcommon-dev libxkbcommon-x11-dev
       - name: Install Conda
-        uses: conda-incubator/setup-miniconda@v2
+        uses: conda-incubator/setup-miniconda@v2.1.1
```

Comment:
Why ping to this specific version? Shouldn't be better to use `v2` to bring the latest 2.x.x version of this action?